extensions/2.0/Vendor/EXT_materials_diffuse_roughness/schema/glTF.EXT_materials_diffuse_roughness.schema.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "EXT_materials_diffuse_roughness glTF extension",
+    "type": "object",
+    "description": "glTF extension that defines roughness of the base diffuse surface.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "diffuseRoughnessFactor": {
+            "type": "number",
+            "description": "The base, diffuse layer's roughness.",
+            "default": 1.0,
+            "minimum": 0.0,
+            "maximum": 1.0,
+            "gltf_detailedDescription": "The base, diffuse layer's roughness."
+        },
+        "diffuseRoughnessTexture": {
+            "allOf": [ { "$ref": "textureInfo.schema.json" } ],
+            "description": "A greyscale texture, representing the diffuse layer's roughness.",
+            "gltf_detailedDescription": "A greyscale texture, representing the diffuse layer's roughness. The value will modulate the diffuseRoughnessFactor."
```

Comment:
Instead of "greyscale" (which is ambiguous and not used in glTF specs) please specify an exact channel (likely red) from which the values are sourced.

This should also be mentioned in README for better visibility.

Reply:
Example:
- https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_materials_transmission#properties
- https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_transmission/schema/material.KHR_materials_transmission.schema.json#L18

Reply:
We should decide which channel though. Does it make sense to pack this channel with other textures?